client/app.js

```diff
@@ -1,15 +1,20 @@
 import React from 'react'
-
-import {Navbar} from './components'
-import {ScreensMap} from './screens'
+import {
+  ScreensNavbar,
+  ScreensMap,
+  ScreenRankingTabs,
```

Comment:
Can we make sure all the displayed components are spelled `Screens` instead of `Screen`? That's the way the folder is spelled c:

Reply:
good catch. Fixed